### PR TITLE
S2060

### PR DESCRIPTION
**Pull request by andamian**

### CODE REVIEW

File: cadcdata/setup.py [hunk 1]
```diff
@@ -31,53 +27,28 @@
...
-# invoking any other functionality from distutils since it can potentially
-# modify distutils' behavior.
-cmdclassd = register_commands(PACKAGENAME, VERSION, RELEASE)
-
-# Freeze build information in version.py
-generate_version_py(PACKAGENAME, VERSION, RELEASE,
-                    get_debug_option(PACKAGENAME))
+# generate the version file
+with open(os.path.join(PACKAGENAME, 'version.py'), 'w') as f:
+    f.write('version = \'{}\''.format(VERSION))
```
Comment (ijiraq) [inline cadcdata/setup.py]: I would have written this as:

"""'{}'""".format(VERSION)

so not using the \ escape.

File: cadcutils/MANIFEST.in [hunk 2]
```diff
@@ -2,7 +2,6 @@ include README.rst
 include CHANGES.rst
```
Comment (ijiraq) [inline cadcutils/MANIFEST.in]: Isn't there a README.rst file?

File: cadcutils/setup.py [hunk 3]
```diff
@@ -31,56 +27,28 @@
 metadata = dict(conf.items('metadata'))
 
 PACKAGENAME = metadata.get('package_name', 'packagename')
-DESCRIPTION = metadata.get('description', 'Astropy affiliated package')
+DESCRIPTION = metadata.get('description', 'CADC package')
 AUTHOR = metadata.get('author', '')
 AUTHOR_EMAIL = metadata.get('author_email', '')
```
Comment (ijiraq) [inline cadcutils/setup.py]: should we have a default cadc email address here?

File: cadcutils/setup.cfg [hunk 4]
```diff
@@ -31,8 +35,8 @@ url = http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca
 edit_on_github = False
 github_project = opencadc/cadctools 
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
+install_requires = requests lxml>=3.7.0 six
```
Comment (ijiraq) [inline cadcutils/setup.cfg]: the requests version must be restricted or handshake files with SSL.

File: cadcdata/setup.py (hunk 1 above)
Comment (andamian) [inline cadcdata/setup.py]: Kind of hard to keep count of """'.

File: cadcutils/MANIFEST.in (hunk 2 above)
Comment (andamian) [inline cadcutils/MANIFEST.in]: Not in this project. Changed to README.md

File: cadcutils/setup.py (hunk 3 above)
Comment (andamian) [inline cadcutils/setup.py]: Done

File: cadcutils/setup.cfg (hunk 4 above)
Comment (andamian) [inline cadcutils/setup.cfg]: restricted to requests>=2.8,<=2.11.1. I've also removed this dependency from vos.

**Comment by andamian** [comment]
Thanks for the prompt code review.

**Pull request merged**